app/controllers/asset_manager_redirect_controller.rb

```diff
@@ -1,6 +1,10 @@
 class AssetManagerRedirectController < ApplicationController
   def show
-    asset_host = URI.parse(Plek.new.public_asset_host).host
-    redirect_to host: asset_host
```

Comment:
Does this mean currently that if you were trying to access a draft asset you would always get redirected to the live version?

Reply:
If your request made it to this controller, yes; but this controller doesn't match whitehall assets yet: https://github.com/alphagov/whitehall/blob/f35fd6fb76e726fbd5e8e94037807e2af3f1dce8/config/routes.rb#L425-L429